client/components/Users.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {connect} from 'react-redux';
+// import { getUsers } from '../redux/';
+import faker from 'faker';
+
+export class Users extends React.Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            users: [],
+            adminUsers: []
+        }
+        this.handleChecked = this.handleChecked.bind(this)
+        this.handleDeleteUser = this.handleDeleteUser.bind(this)
+        this.handleResetPassword = this.handleResetPassword.bind(this)
+
+    }
+
+    componentDidMount(){
+        // I'll change the code below once we confirm the users data
```

Comment:
What do you mean by confirm the users data? Thinking this should eventually end up in the seed file and this component would make a db call via Redux

Reply:
I meant, I would change it after we have the users data. I wasn't sure if someone else was working on setting up the users data but I can take care of it 🙂